x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/cti/event_enrichment/query.ts

```diff
@@ -5,56 +5,59 @@
  * 2.0.
  */
 
-import type { CtiQueries } from '../../../../../../common/search_strategy/security_solution/cti';
+import type { EventEnrichmentRequestOptions } from '../../../../../../common/api/search_strategy';
 import { createQueryFilterClauses } from '../../../../../utils/build_query';
-import type { SecuritySolutionFactory } from '../../types';
 import { buildIndicatorShouldClauses } from './helpers';
 
-export const buildEventEnrichmentQuery: SecuritySolutionFactory<CtiQueries.eventEnrichment>['buildDsl'] =
-  ({ defaultIndex, eventFields, filterQuery, timerange: { from, to } }) => {
-    const filter = [
-      ...createQueryFilterClauses(filterQuery),
-      { term: { 'event.type': 'indicator' } },
-      {
-        range: {
-          '@timestamp': {
-            gte: from,
-            lte: to,
-            format: 'strict_date_optional_time',
-          },
+export const buildEventEnrichmentQuery = ({
+  defaultIndex,
+  eventFields,
+  filterQuery,
+  timerange: { from, to },
+}: EventEnrichmentRequestOptions) => {
```

Comment:
Fyi, for other reviewers, the only change here was adding the type and formatting